lasair_zooniverse.py

```diff
@@ -178,7 +176,10 @@ def create_subjects_and_link_to_project(self, proto_subjects, project_id, workfl
             for proto_subject in proto_subjects:
                 subject = Subject()
                 subject.links.project = project
-                subject.add_location(proto_subject['location_lc'])
+                subject.locations.append('application/json')
+                json_data = open(proto_subject['location_lc'], 'rb')
+                subject._media_files.append(json_data.read())
+                json_data.close()

```

Comment:
Maybe add a comment here to explain why `subject.add_location` doesn't work (it uploads the JSON string as plain text.)